app/models/maintenance_tasks/run.rb

```diff
@@ -206,6 +208,21 @@ def csv_attachment_presence
       nil
     end
 
+    # Performs validation on the parameters to use for the Task. Attributes are
+    # assigned to a Task instance based on the parameters passed to the Run's
+    # constructor. If the Task is invalid, the errors are added to the Run.
+    def valid_task_parameters?
+      if task.invalid?
+        error_messages = task.errors.full_messages
+        errors.add(
+          :parameters,
+          "are invalid: #{error_messages.join("; ")}"
```

Comment:
I found that the default error messages weren't great because of the call to `humanize`:
<img width="538" alt="image" src="https://user-images.githubusercontent.com/3535/119005247-483d3500-b95d-11eb-8c75-9c613a569c34.png">

Reply:
Do you have suggestions for a better message? 😄